### ISSUE TITLE
Problem trying to run the lint script

### ISSUE THREAD

**Issue by wallergoble** [moment/luxon#342]
I pulled down a fresh copy of master and tried to run `npm run lint`.

I got the following error: 
```
luxon git:(master) npm run lint

> luxon@1.4.3 lint /Users/wwg6cou/github/wallergoble/luxon
> eslint --quiet src test benchmarks

TypeError: Cannot read property 'sync' of undefined
    at Program (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js:166:59)
    at listeners.(anonymous function).forEach.listener (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/util/safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/util/node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/util/node-event-generator.js:280:22)
    at NodeEventGenerator.enterNode (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/util/node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:632:23)
    at nodeQueue.forEach.traversalInfo (/Users/wwg6cou/github/wallergoble/luxon/node_modules/eslint/lib/linter.js:860:28)
    at Array.forEach (<anonymous>)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! luxon@1.4.3 lint: `eslint --quiet src test benchmarks`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the luxon@1.4.3 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Am I missing something? I googled around and people say it's because their prettier version is < 1.7.0 but the package.json says it's 1.7.4 here. 

Can anyone help?

**Comment by wallergoble** [comment]
Realized the build is just failing. Trying to dive into why.

**Comment by icambron** [comment]
Fixed for now in 55eccc526
